packages/flutter/test/widgets/actions_test.dart

```diff
@@ -392,9 +393,11 @@ void main() {
       await buildTest(true);
       expect(hovering, isFalse);
       expect(focusing, isFalse);
+
+      focusNode.dispose();
```

Comment:
Can you update disposals to be aligned with newly gained knowledge: https://github.com/dart-lang/leak_tracker/pull/138

Reply:
@polina-c great rule, thanks.